### DISCUSSION TITLE
Automated Literature Review

### DISCUSSION THREAD

**Discussion by mrchristian** [semanticClimate/ipcc-report-cataloguing#9] [Ideas]
## Project link

 👉 [Automated Literature Review](https://github.com/semanticClimate/automated-literature-review/blob/main/README.md)

### Outline of the documentation project

- Goal: Create a guide and accompanying Jupyter Notebook for a generic Automated Litertaure Review using the Ami software
- Duration: 12 weeks, starting April 2, 2024
- Participants: Project lead @mrchristian (Simon) | 3 Interns from NIPGR | input Team semanticClimate
- Reporting: @flower1430 (Renu) NIPGR Intern administration, @gilienv (Gita) Lab Director, NIPGR
 
#### Week 16: 2-5 April

1. Orientation
2. Internship administration
3. Write project description
4. Set out individual questions and main question for lit searches
5. Identify documentation methods, review process
6. Agree with team semanticClimate what main parts are needed in Notebooks and workflow

**Comment by mrchristian** [comment]
Resources: https://docs.google.com/spreadsheets/d/1EwRFP3eegarg6rJZAEbE1g360VNKGWYGiYPKn5ci2Vo/edit#gid=0

**Comment by mrchristian** [comment]
Key links:

- Documentation: https://pygetpapers.readthedocs.io/en/latest/#
- Notebooks: 
- https://colab.research.google.com/drive/1QhkUNom8U_BR1bRVNNn6wWsIdtRmpxQt?usp=sharing#scrollTo=ZfwlxRB8gG5I
- https://colab.research.google.com/drive/12ixmez8zh288hBGzwWmBaj29leeuQRD7#scrollTo=AXDshvzTef_m

**Comment by mrchristian** [comment]
Example project description: https://github.com/NFDI4Culture/CPS-Demo/blob/main/README.md

**Comment by petermr** [comment]
This is really great Simon,
When you have "finished" it will be useful to have a single entry point so
non-attenders can look inn and see what you are all doing.

P.


On Tue, Apr 2, 2024 at 11:40 AM Simon Worthington ***@***.***>
wrote:

> Example project description:
> https://github.com/NFDI4Culture/CPS-Demo/blob/main/README.md
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/semanticClimate/discussions/9#discussioncomment-8982340>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCS6KF64TAVSCORGKYM3Y3KDIDAVCNFSM6AAAAABFTBIBUSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DSOBSGM2DA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID:
> <semanticClimate/ipcc-report-cataloguing/repo-discussions/9/comments/8982340
> @github.com>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by mrchristian** [comment]
Covered in the intern meetings so far - 8 Mar 2024:

- Introduction to software citation
- Use of citation manager Zotero and CSL
- Introduction to making literature reviews: Guides to academic literature reviews; How to write a literature review question; Example papers of literature review
- Use of Colab Jupyter Notebooks
- Using Lab Notebook to record use of Colab Notebook
- Intro to academic writing style guides, and technical writing style guides
- Noting terms used in a project (working nomenclature)
